NEWS

```diff
@@ -1,7 +1,9 @@
 NEWS
 
-GRASS GIS 7
+GRASS GIS 8
+ o https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures80
```

Comment:
Grass7 => Grass8?

Reply:
Good catch, fixed in 8078cc676df18473fc3aceee6b9f7a78822b46e9